reference/SUPPORTED_QUERIES.md

```diff
@@ -0,0 +1,159 @@
+
+```python
+Customer.objects.filter(
+    Q(
+        CsMatch(
+            CsMatchV1(F("name")),
+            CsMatchV1(
+                Value(EqlText("ali", "customers", "name").to_db_format("match"))
+            )
+        )
+    )
+)
+```
+
+This is the more expressive but verbose method of expressing EQL queries with Django ORM.
+
+The following table shows the relevant classes in `eqldjango`.
+
+| Function or operator class | PostgreSQL function or operator |
+|----------------------------|---------------------------------|
+| CsMatchV1                  | cs_match_v1                     |
+| CsUniqueV1                 | cs_unique_v1                    |
+| CsOre648V1                 | cs_ore_64_8_v1                  |
+| CsSteVecV1                 | cs_ste_vec_v1                   |
+| CsSteVecValueV1            | cs_ste_vec_value_v1             |
+| CsSteVecTermV1             | cs_ste_vec_term_v1              |
+| CsGroupedValueV1           | cs_grouped_value_v1             |
+| CsContains                 | @>                              |
+| CsMatch                    | @>                              |
+| CsContainedBy              | <@                              |
+| CsEquals                   | =                               |
+| CsGt                       | >                               |
+| CsLt                       | <                               |
+
+### SQLAlchemy
+
+For SQLAlchemy, `eqlpy.eqlalchemy` provides python functions that correspond to the EQL functions.
+`eqlpy` also provides EQL-specific type decorators which helps with converting between the database types and Python types.
+
+```python
+session.query(Example)
+    .filter(
+        cs_match_v1(Example.encrypted_utf8_str).op("@>")(
+            cs_match_v1(
+                EqlText("hello", "examples", "encrypted_utf8_str").to_db_format(
+                    "match"
+                )
+            )
+        )
+    )
+    .one()
+```
+
+With those functions, instead of calls to `cs_*` functions directly in SQL, they can be expressed in Python.
+
+The following EQL functions are available in Python for SQLAlchemy.
+
+- cs_unique_v1
+- cs_match_v1
+- cs_ore_64_8_v1
+- cs_ste_vec_v1
+- cs_ste_vec_value_v1
+- cs_ste_vec_term_v1
+- cs_grouped_value_v1
+
+The EQL-specific type decorators are:
+
+- EncryptedInt
+- EncryptedBoolean
+- EncryptedDate
+- EncryptedFloat
+- EncryptedUtf8Str
+- EncryptedJsonb
+
+### psycopg
+
+For psycopg, the query interface involves SQL, with `cs_*` functions, with EQL value types:
+
+```python
+cur.execute(
+    "SELECT * FROM examples WHERE cs_match_v1(encrypted_utf8_str) @> cs_match_v1(%s)",
+    (EqlText("hello", "examples", "encrypted_utf8_str").to_db_format("match"),),
+)
+```
+
+This is the more verbose but expressive way of writing EQL queries.
+
+`eqlpy` provides the following classes to represent EQL values:
+
+- EqlInt
+- EqlBool
+- EqlDate
+- EqlFloat
+- EqlText
+- EqlJsonb
```

Comment:
You could link to the relevant sections in `reference/VALUE_CLASSES.md`.

Reply:
done